hypothesis-python/src/hypothesis/extra/pytestplugin.py

```diff
@@ -128,12 +132,34 @@ def note_statistics(stats):
             gathered_statistics[item.nodeid] = lines
             item.hypothesis_statistics = lines
 
+        # Peturb PRNG so that we can detect e.g. everything calling seed(0)
+        random.seed(random.randrange(2 ** 32))
+        before = hash(random.getstate())
```

Comment:
Why hash as opposed to use the whole value? It's unlikely to be any faster, and Python hashes tend to be fast rather than low collision rate.

Reply:
Shorter logs, and I didn't expect it to have much impact... will revert though.

Reply:
Shorter logs where? You're not printing the state anywhere are you? I don't think there's much need to - it doesn't convey information other than the fact that they're the same.

Reply:
No, no, I was just printing everywhere (:disappointed:) while working out where to hook into Pytest.  All gone now that I have nice regression tests instead!